File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Open and save Word document in ASP.NET MVC | Syncfusion
+description: Open and save Word document in ASP.NET MVC application using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Open and Save Word document in ASP.NET MVC
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, and edit **Word** documents programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in ASP.NET MVC**.
+
+## Steps to open and save Word document programmatically:
+
+Step 1: Create a new ASP.NET MVC application project.
+
+Step 2: Install the [Syncfusion.DocIO.AspNet.Mvc5](https://www.nuget.org/packages/Syncfusion.DocIO.AspNet.Mvc5) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 3: Include the following namespace in that HomeController.cs file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
```

Comment:
Mention the DocIO related namespaces alone

using Syncfusion.DocIO;
using Syncfusion.DocIO.DLS;